source/utilities.tex

```diff
@@ -14990,12 +15034,12 @@
 \end{note}
 
 \pnum
-For a string in a Unicode encoding,
-implementations should estimate the width of a string
-as the sum of estimated widths of
-the first code points in its extended grapheme clusters.
-The extended grapheme clusters of a string are defined by \UAX{29}.
-The estimated width of the following code points is 2:
+For a sequence of characters in a Unicode encoding,
```

Comment:
Are the following changes, which the paper for #6126 made for existing wording, not required here?
```suggestion
For a sequence of characters in UTF-8, UTF-16, or UTF-32,
```